composer.json

```diff
@@ -14,13 +14,22 @@
         }
     },
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
+        "aerospike/aerospike-client-php": "^7.0",
+        "symfony/cache": "^4.2"
     },
     "require-dev": {
         "lmc/coding-standard": "^1.3",
         "phpstan/phpstan-shim": "^0.11",
         "phpunit/phpunit": "^7.5"
     },
+    "suggest": {
+        "doctrine/cache": "Allows use of the DoctrineSymfonyAdapter for the AerospikeCache compatibility with Doctrine Cache"
```

Comment:
I forgot to mention it previously, sorry :speak_no_evil:  - `doctrine/cache` should be added as dev dependency (it is required for phpstan and unit tests).

Reply:
Done